databases/installing-postgresql.md

```diff
@@ -19,6 +19,8 @@
 
 ### 1. Install PostgreSQL through app [On Windows].
 
+* **Reference**: [guru99.com](https://www.guru99.com/download-install-postgresql.html)
```

Comment:
Is just part one from guru 99, or the whole Lesson section?  If its the whole lesson section, move the reference to line 19.

Reply:
Part 1 only

Reply:
Does part 2 need a reference?

Reply:
No